buildscripts/ci/linux/setup.sh

```
@@ -41,231 +42,4 @@ while [[ "$#" -gt 0 ]]; do
-  cd $origin_dir
-fi
-
-##########################################################################
-# POST INSTALL
-##########################################################################
-
-chmod +x "$ENV_FILE"
-
-df -h .
-echo "Setup script done"
+bash $MF_DIR/buildscripts/ci/linux/setup.sh --arch $PACKARCH --compiler $COMPILER --build-pipewire $BUILD_PIPEWIRE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)